diagnostic.md

```diff
@@ -33,7 +47,11 @@ y = 10
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
 <!-- solution below -->
+x = 3
+
+y = 10
 
+The variables don't change but their values do.
```

Comment:
Hey Jim we wanted you to note on all four lines what the variables values were but looks like you knew what they were so good job.